erpnext/selling/page/point_of_sale/point_of_sale.js

```diff
@@ -922,7 +939,25 @@ class POSCart {
 		this.$qty_total.find('.quantity-total').text(total_item_qty);
 		this.frm.set_value("pos_total_qty",total_item_qty);
 	}
-
+	make_order_type_options()
+	{
+		this.order_type_field = frappe.ui.form.make_control({
+			df: {
+				fieldtype: 'Select',
+				label: 'Order Type',
+				fieldname: 'new_order_type',
+				options: cur_frm.get_field("new_order_type").df.options,
+				reqd: 1,
+				onchange: () => {
+					this.events.on_order_type_change(this.order_type_field.get_value());
+					this.events.get_loyalty_details();
+				}
+			},
+			parent: this.wrapper.find('.customer-field'),
+			render_input: true
+		});
+		this.order_type_field.set_value(this.frm.doc.new_order_type);
```

Comment:
This doesn't get called on refresh, right? If you're trying to set a default value, this can be done with a `default` key in the `df` definition above